tools/build_chain.sh

```diff
@@ -570,6 +565,21 @@ generate_config_ini()
     ; when the outgoing bandwidth exceeds the limit, the block synchronization operation will not proceed
     ;outgoing_bandwidth_limit=2
 EOF
+
+if [ -z "${guomi_mode}" ]; then
+    cat << EOF >> ${output}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5484572)